docs/process-based-load-script.md

```diff
@@ -0,0 +1,114 @@
+# Processes based load script
+
+* [Requirements](#requirements)
+* [Parameters description](#parameters-description)
+* [Examples](#examples)
+
+## Requirements
+
+* native libs
+    * libindy >= 1.4.0
+
+* python lib
+    * python3-indy >= 1.4.0
```

Comment:
Load script will not work with libindy 1.4.0 (stable). It works with libindy >= 1.4.0~588 (master) and python3-indy >= 1.4.0.dev588.

Reply:
fixed